src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -51,11 +51,15 @@ public InputOutputColumnPair(string outputColumnName, string inputColumnName = n
     public static class TransformExtensionsCatalog
     {
         /// <summary>
-        /// Copies the input column to another column named as specified in <paramref name="outputColumnName"/>.
+        /// Create a <see cref="ColumnCopyingEstimator"/>, which copies the data from the column specified in <paramref name="inputColumnName"/>
+        /// to a new column: <paramref name="outputColumnName"/>.
```

Comment:
Maybe
```suggestion
        /// to a new column <paramref name="outputColumnName"/>.
```

Reply:
Just remove the:?  I think i might leave it, since Natalie seems ok with it?

---
In reply to: [275449364](https://github.com/dotnet/machinelearning/pull/3316#discussion_r275449364) [](ancestors = 275449364)